src/test/java/com/commercetools/sync/inventory/utils/InventorySyncUtilsTest.java

```diff
@@ -0,0 +1,148 @@
+
+    private final InventoryEntry inventoryEntry;
+    private final InventoryEntry inventoryEntryWithCustomField1;
+    private final InventoryEntryDraft similarDraft;
+    private final InventoryEntryDraft variousDraft;
+
+    {
+        inventoryEntry = InventoryEntryMock.of("123", 10l, 10, DATE_1)
+                .withChannelRefExpanded("111", "key1")
+                .build();
+        inventoryEntryWithCustomField1 = InventoryEntryMock.of("123", 10l, 10, DATE_1)
+                .withChannelRefExpanded("111", "key1")
+                .withCustomFieldExpanded(CUSTOM_TYPE_ID, CUSTOM_TYPE_KEY, CUSTOM_FIELD_1_NAME, CUSTOM_FIELD_1_VALUE)
+                .build();
+
+        similarDraft = InventoryEntryDraftBuilder.of("123", 10l, DATE_1, 10, Channel.referenceOfId("111")).build();
+        variousDraft = InventoryEntryDraftBuilder.of("321", 20l, DATE_2, 20, Channel.referenceOfId("222")).build();
+    }
+
+    @Test
+    public void buildActions_WithSimilarEntries_ShouldReturnEmptyList() {
+        List<UpdateAction<InventoryEntry>> actions = InventorySyncUtils
+                .buildActions(inventoryEntry, similarDraft, InventorySyncOptionsBuilder.of(mock(CtpClient.class))
+                        .build(), mockTypeService());
+
+        assertThat(actions).isEmpty();
+    }
+
+    @Test
+    public void buildActions_WithVariousEntries_ShouldReturnActions() {
+        List<UpdateAction<InventoryEntry>> actions = InventorySyncUtils
+                .buildActions(inventoryEntry, variousDraft, InventorySyncOptionsBuilder.of(mock(CtpClient.class))
+                        .build(), mockTypeService());
+
+        assertThat(actions).hasSize(4);
+    }
+
+    @Test
+    public void buildActions_WithSimilarEntriesAndSameCustomFields_ShouldReturnEmptyList() {
+        final InventoryEntryDraft draft = InventoryEntryDraftBuilder.of(similarDraft)
+                .custom(getDraftOfCustomField(CUSTOM_FIELD_1_NAME, CUSTOM_FIELD_1_VALUE))
+                .build();
+        List<UpdateAction<InventoryEntry>> actions = InventorySyncUtils
+                .buildActions(inventoryEntryWithCustomField1, draft, InventorySyncOptionsBuilder
+                                .of(mock(CtpClient.class))
+                                .build(),
+                        mockTypeService());
+
+        assertThat(actions).isEmpty();
+    }
+
+    @Test
+    public void buildActions_WithSimilarEntriesAndNewCustomTypeSet_ShouldReturnActions() {
+        final InventoryEntryDraft draft = InventoryEntryDraftBuilder.of(similarDraft)
+                .custom(getDraftOfCustomField(CUSTOM_FIELD_2_NAME, CUSTOM_FIELD_2_VALUE))
+                .build();
+        List<UpdateAction<InventoryEntry>> actions = InventorySyncUtils
+                .buildActions(inventoryEntry, draft, InventorySyncOptionsBuilder.of(mock(CtpClient.class)).build(),
+                        mockTypeService());
+
+        assertThat(actions).hasSize(1);
```

Comment:
guess `actions.get(0)` properties also should be asserted, isn't it? (same for the rest of methods)